### PR TITLE
Minor fixes and updates to logging related to multinet

### PR DESCRIPTION
**Pull request by mmamczur**
Fix some missing values

also add information about the network used when creating NEG

### CODE REVIEW

File: pkg/backends/backends.go [hunk 1]
```diff
@@ -352,35 +352,35 @@ func (b *Backends) EnsureL4BackendService(name, hcLink, protocol, sessionAffinit
 
 	// Create backend service if none was found
 	if bs == nil {
-		klog.V(2).Infof("EnsureL4BackendService(%v, _, %v, _, %v, %v ...): creating backend service %v", name, protocol, scheme, namespacedName)
+		klog.V(2).Infof("EnsureL4BackendService(%v, _, %v, _, %v, %v ...): creating backend service %v", name, protocol, scheme, namespacedName, namespacedName.String())
```
Comment (cezarygerard) [inline pkg/backends/backends.go:355]: klog.V(2).Infof("EnsureL4BackendService(%v, _, %v, _, %v, %v ...): creating backend service", name, protocol, scheme, namespacedName)

originally backend name was the only logged var, now it's a first arg

[same applies to the rest of changes in this file]

Comment (mmamczur) [reply pkg/backends/backends.go]: ouh, I somehow this was supposed to be the service name. changed

**Pull request merged**